recipes/llvm-core/all/cmake/FindZLIB.cmake

```diff
@@ -0,0 +1,22 @@
+# The FindZLIB find module provided by CMake behaves differently than the FindZLIB.cmake script generated by CONAN.
+# This script provides the needed variables and properties needed by LLVM.
+#
+# Specifically LLVM relies on:
+# - ZLIB_LIBRARY variable being set
+# - LOCATION property set in the ZLIB::ZLIB target
+
+if(ZLIB_FIND_REQUIRED)
+  list(APPEND OPTIONS REQUIRED)
+endif()
+if(ZLIB_FIND_QUIET)
+  list(APPEND OPTIONS QUIET)
+endif()
+
+list(REMOVE_ITEM CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR})
+find_package(ZLIB ${OPTIONS})
+
+if (ZLIB_FOUND)
+  message(STATUS "Setting ZLIB_LIBRARY which is needed by llvm-core")
+  list(GET ZLIB_LIBRARIES 0 ZLIB_LIBRARY)
```

Comment:
Could this be in the root cmake wraper?

We often would just call find package and setup the properties or aliases instead of custom modules

Reply:
Yes, that worked. Thank you for the tip!